interceptor/tracing_interceptor.go

```diff
@@ -27,11 +27,13 @@ import (
 	"fmt"
 	"time"
 
+	"github.com/nexus-rpc/sdk-go/nexus"
 	commonpb "go.temporal.io/api/common/v1"
 
 	"go.temporal.io/sdk/activity"
 	"go.temporal.io/sdk/client"
 	"go.temporal.io/sdk/converter"
+	"go.temporal.io/sdk/internal"
```

Comment:
The tracing_interceptor shouldn't need to depend on internal

Reply:
Ah this was autocomplete by the IDE, I removed the direct dependency.
Just note that it doesn't matter since the interceptor package does depend on internal, it aliases the internal types for export.

Reply:
The tracing interceptor should depend on the aliased types, there are still references to the internal types here